code/src/components/Intro.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+
+export const Intro = () => {
```

Comment:
Nice, small and clean component. Remember, you should still use semantic html elements. Maybe section or article is a better element than div